manifests/cluster-rbac/server/argocd-server-clusterrole.yaml

```diff
@@ -36,3 +36,15 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - create   # supports triggering jobs from UI
+- apiGroups:
+  - argoproj.io
+  resources:
+  - workflows
+  verbs:
+  - create   # supports triggering workflows from UI
```

Comment:
Should this be in a separate PR? Or maybe update PR title and description to include this change

Reply:
Sure it makes sense. I will update the description and title shortly.

Reply:
@terrytangyuan commit and description updated. Thanks!